qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddPersonPage.java

```diff
@@ -14,31 +14,31 @@
 
 public class AddPersonPage extends Page {
 
-	private static final By PERSON_NAME = By.id("personName");
-	private static final By BIRTHDATE = By.id("birthdate");
-	private static final By AGE = By.id("age");
+	private static final By PERSON_NAME_FIELD = By.id("personName");
```

Comment:
Please use the default element attribute description
```suggestion
	private static final By PERSON_NAME= By.id("personName");
	private static final By BIRTHDATE = By.id("birthdate");
	private static final By AGE = By.id("age");
```

Reply:
Those are more readable than the current changes.